brave/src/main/java/brave/CurrentSpanCustomizer.java

```diff
@@ -0,0 +1,56 @@
+package brave;
+
+/**
+ * Provides a mechanism for end users to be able to customise the current span.
+ *
+ * Handles the case of there being no current span in scope.
+ */
+public class CurrentSpanCustomizer implements SpanCustomizer {
+
+  private Tracing tracing;
```

Comment:
I'd make this ref final and to Tracer (so that each call doesn't need to do tracing.tracer)